.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+python:
+    - "2.7"
```

Comment:
@clintonb I've Python 3 to the tests, but it seems like Django gives different error messages depending on which version it's running. I suspect it has to do with optparse (old) versus argparse (new), but I haven't tracked it down just yet.